lib/nunjucks-html-loader.js

```diff
@@ -48,6 +48,7 @@ function buildSiteMap(context) {
     const group = frontmatter.group || '';
     const lang = frontmatter.lang || '';
     const experimental = frontmatter.experimental || '';
+    const topLevel = frontmatter.topLevel || '';
```

Comment:
Could possibly remove the `topLevel` code completely and check if a `parent` exists and what it is. Should be able to implement something like `{% if pageInfo.parent[0].title === 'home' %}` in the template.

Reply:
Cheers.  I was hoping there would be a better way in the templating.  Will take a look

Reply:
Should save the overhead of remembering to add frontmatter

Reply:
Yep deffo.  Worked a treat 👍